### DISCUSSION TITLE
Handle routes by myself

### DISCUSSION THREAD

**Discussion by carloschneider** [i18next/next-i18next#2296] [Q&A]
I would like to manually create the routes in NextJS `/en`, `/es`, `/pt` and send the current language to next-i18next.

Is there any manual way to tell next-i18next (appWithTranslation) which language should be loaded?

**Comment by adrai** [comment]
The only way is to preload the translations and pass them via resources option here: https://github.com/i18next/next-i18next/blob/master/examples/simple/next-i18next.config.js#L27
